holoviews/plotting/bokeh/element.py

```diff
@@ -608,14 +629,20 @@ def _init_plot(self, key, element, plots, ranges=None):
             # are not really an issue
             warnings.simplefilter('ignore', UserWarning)
             fig = figure(title=title, **properties)
+        fig.xaxis[0].update(**axis_props['x'])
+        fig.yaxis[0].update(**axis_props['y'])
```

Comment:
Shame the Bokeh API doesn't let you set this up once together with the figure...